spec/std/big/big_decimal_spec.cr

```diff
@@ -435,6 +449,9 @@ describe BigDecimal do
 
     it { -2.01.to_big_d.ceil.should eq(-2) }
     it { -2.91.to_big_d.ceil.should eq(-2) }
+
+    it { "-123.000".to_big_d.ceil.should eq(-123) }
```

Comment:
These specs pass on `0.32.1` because comparison applies the correct scale. That's why I asked very tentatively whether there's a way to spec this because it needs to look at internal state.

The specs are probably good, still but they don't test what they're supposed to.

Reply:
You're right. I've changed it to compare the `#value` instead.